ocaml/xenopsd/lib/bootloader.ml

```diff
@@ -205,9 +187,7 @@ let sanity_check_path p =
       let canonical_path = Xapi_stdext_unix.Unixext.resolve_dot_and_dotdot p in
       match Filename.dirname canonical_path with
       | "/var/run/xen/pygrub"
-      (* From pygrub, including when called by eliloader *)
-      | "/var/run/xend/boot" (* From eliloader *) ->
-          canonical_path
```

Comment:
Careful, now pygrub will raise an error without:
```suggestion
      (* From pygrub *)
      ->
          canonical_path
```

Reply:
We've dropped `xend` from the product long ago, I think that was a toolstack before `xl`, there are 2 lines there with pygrub, I only dropped one.

Reply:
Ah I see what you mean

Reply:
Fixed in latest push.